app/code/Magento/InventoryCatalog/composer.json

```diff
@@ -13,7 +13,8 @@
     "magento/module-inventory-configuration-api": "*",
     "magento/module-inventory-indexer": "*",
     "magento/module-inventory-sales-api": "*",
-    "magento/module-inventory-configuration": "*"
+    "magento/module-inventory-configuration": "*",
+    "magento/module-asynchronous-operations": "*"
```

Comment:
Let's move the whole Synchronization logic to the new Module , that would help us to keep all the dependencies clean